Lib/struct.py

```diff
@@ -0,0 +1,15 @@
+__all__ = [
+    # Functions
+    'calcsize', 'pack', 'pack_into', 'unpack', 'unpack_from',
```

Comment:
Note, we do not require this module, since we have one written in rust.

Reply:
Whoops, I thought I had removed it.